init.go

```diff
@@ -111,6 +111,7 @@ func Transport(baseURL *url.URL, insecureTLS bool) *httptransport.Runtime {
 func init() {
 	seed := time.Now().UnixNano()
 	rand.Seed(seed)
+	gofakeit.SetGlobalFaker(gofakeit.NewCustom(NewConcurrentRand(seed)))
```

Comment:
`Faker` is already thread-safe, so it can be:

```suggestion
	gofakeit.SetGlobalFaker(gofakeit.New(seed))
```

Reply:
It also resolves all comments issues  🙂

Reply:
doesn't seems so https://github.com/brianvoe/gofakeit/issues/32